web/client/components/data/query/CrossLayerFilter.jsx

```diff
@@ -47,6 +48,16 @@ export default ({
         index: 0
     }]} = queryCollection;
 
+    const UnMatchingLayerOptions = layers
+        .filter( l => !isSameOGCServiceRoot(searchUrl, l));
+
+    const renderUnMatchingLayersInfo = () => {
+        if (UnMatchingLayerOptions.length) {
+            return (<ToolbarButton className="square-button-md no-border unmatching-sources-info-icon"
```

Comment:
You can use InfoPopover ( or `DisposablePopover` in case of issues with position and scroll). 
See `ChartAdvancedOptions.jsx` for an example of usage.
```suggestion
                        return (<InfoPopover bsStyle="link" text={<Message msgId="queryform.crossLayerFilter.errors.layersExcluded" />}/>);

```